### DISCUSSION TITLE
Can music be in */media, but seperate folders?

### DISCUSSION THREAD

**Discussion by mikestudent** [AzuraCast/AzuraCast#6935] [Q&A]
Purpose is to preserve album art, right now I have one artist playing with a totally different artist's album art.  I've not tried it yet, finally got everything working.  In other words, I don't have the balls to wreck something that's working.

**Comment by BusterNeece** [comment]
I mean, you absolutely can have subfolders and put your media in those folders. That's strongly recommended if you're uploading media with album art files, as we look for album art on a per-folder level to apply in the absence of per-file album art.
